Celeste.Mod.mm/Mod/UI/OuiModToggler.cs

```diff
@@ -472,9 +643,14 @@ class OuiModToggler : OuiGenericMenu, OuiModOptions.ISubmenu {
             restartMessage1 = null;
             restartMessage2 = null;
             modYamls = null;
+            modFilename = null;
             modToggles = null;
             modLoadingTask = null;
             toggleDependencies = true;
+            protectFavorites = false;
```

Comment:
```suggestion
            protectFavorites = true;
```
nitpick since it makes protectFavorites enabled on first entry then disabled on the next ones